README.md

```diff
@@ -300,25 +300,24 @@ Let's create a simple app that integrates PSPDFKit and uses the react-native-psp
  }
 ```
 
-8. PSPDFKit targets modern platforms, so you'll have to set the `minSdkVersion` to 19. In `YourApp/android/build.gradle` (note **two** places to edit):
+8. PSPDFKit targets modern platforms, so you'll have to set the `minSdkVersion` to 19. In `YourApp/android/build.gradle`:
 ```diff
 ...
-  ext {
--     buildToolsVersion = "28.0.2"
-+     buildToolsVersion = "28.0.3"
--     minSdkVersion = 16
-+     minSdkVersion = 19
-      compileSdkVersion = 28
-      targetSdkVersion = 27
-      supportLibVersion = "28.0.0"
-  }
+ buildscript {
+     ext {
+         buildToolsVersion = "28.0.3"
+-        minSdkVersion = 16
++        minSdkVersion = 19
+         compileSdkVersion = 28
+         targetSdkVersion = 28
+         supportLibVersion = "28.0.0"
 ...
 ```
  
-9. We will also need to enable MultiDex and Java 8 support. In `YourApp/android/app/build.gradle` (note **two** places to edit):
+9. We will also need to enable MultiDex support. In `YourApp/android/app/build.gradle`:
```

Comment:
Why is enabling Java 8 no longer required?

Reply:
Is required but already present.